resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -956,6 +1075,21 @@ public class ApiClient implements AutoCloseable {
             return this;
         }
 
+        public Builder withLogger(Logger logger) {
```

Comment:
Probably best to leave this out and just let them use withLoggingConfiguration, unless you can think of a good reason for it.